src/markdown-pages/build-apps/customize-open-source-visualization.mdx

```diff
@@ -0,0 +1,1141 @@
+                  />
+                  <VictoryAnimation duration={1000} data={percent}>
+                    {(percent) => (
+                      <VictoryLabel
+                        textAnchor="middle"
+                        verticalAnchor="middle"
+                        x={CHART_WIDTH / 2}
+                        y={CHART_HEIGHT / 2}
+                        text={`${Math.round(percent)}%`}
+                        style={{ ...baseLabelStyles, fontSize: 45 }}
+                      />
+                    )}
+                  </VictoryAnimation>
+                  <VictoryLabel
+                    text={label}
+                    lineHeight={1}
+                    x={CHART_WIDTH / 2}
+                    y={BOUNDS.Y - LABEL_SIZE}
+                    textAnchor="middle"
+                    style={{ ...baseLabelStyles, fontSize: LABEL_SIZE }}
+                  />
+                </svg>
+              );
+            }}
+          </NrqlQuery>
+        )}
+      </AutoSizer>
+    );
+  }
+}
+
+const EmptyState = () => (
+  <Card className="EmptyState">
+    <CardBody className="EmptyState-cardBody">
+      <HeadingText
+        spacingType={[HeadingText.SPACING_TYPE.LARGE]}
+        type={HeadingText.TYPE.HEADING_3}
+      >
+        Please provide a NRQL query & account ID pair
+      </HeadingText>
+      <HeadingText
+        spacingType={[HeadingText.SPACING_TYPE.MEDIUM]}
+        type={HeadingText.TYPE.HEADING_4}
+      >
+        This Visualization supports NRQL queries with a single SELECT clause
+        returning a percentage value (0 to 100 rather than 0 to 1). For example:
+      </HeadingText>
+      <code>
+        {'FROM Transaction SELECT percentage(count(*), WHERE duration < 0.1)'}
+      </code>
+    </CardBody>
+  </Card>
+);
+```
+
+First, you created a new constant, called `colorScale`, which is an array of the `segmentColor` values from `this.props.colors`. Then, you set the `VictoryPie` component's `colorScale` prop. Finall, you removed `VictoryPie.style` because the colors are now controlled by `colorScale`.
+
+</Step>
+
+<Step>
+
+From your Nerdpack's root directory, run a local server:
+
+```sh
+nr1 nerdpack:serve
+```
+
+Once the server is running, find the url for your local `circular-progress-bar`:
+
+```sh
+[output] {purple}Visualizations:
+[output]  {muted}⁎  {green}circular-progress-bar {blue}https://one.nr/04ERPALBYjW
+[output]  {muted}⁎  {green}range-chart {blue}https://one.nr/0oqQaxezJj1
+[output]  {muted}⁎  {green}stacked-bar-chart {blue}https://one.nr/0PLRElq3bwa
+```
+
+</Step>
+
+<Step>
+
+Open your locally served visualization and configure your chart with your account, data query, and segment colors:
+
+![Progress bar with custom colors](../../images/build-an-app/progress-bar-chart-with-colors.png)
+
+Because there are two segments, you add two colors. The first color is for the progress section. The second color is for the remaining percentage.
```

Comment:
Do you think we should call out the plus button here to add a second color? This is a nice example of using a `collection`, but I wonder if folks would see the single color field and not know how to get the second field. 

Feel free to disregard if you think the plus button in the UI would be self explanatory. I just don't know if a new user following this guide would know to click there. Could be worth a mention